reports/get_stats.rmd

```diff
@@ -0,0 +1,310 @@
+---
+title: "Youthrex data table script"
+author: "Ben Brew"
+output:
+    html_document:
+      toc: true
+      theme: united
+---
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, comment=NA, error=FALSE}
+# no scientific notation
+options(scipen=999)
+library(tidyverse)
+library(dplyr)
+library(knitr)
+
+
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, comment=NA, error=FALSE}
+##########
+# read in and finish cleaning census data 
+##########
+dat <- read_feather('data/census_all.feather')
```

Comment:
I don't have a census_all.feather on this branch, only a census.feather. 

When I replaced it with census.feather, it ran, but the markdown just had a few lines and no tables.

But maybe this isnt the script I should be looking at?